docker/helm/README.md

```diff
@@ -0,0 +1,13 @@
+
+# Fluss Helm Chart
+
+This chart deploys a Fluss cluster with a Coordinator and a StatefulSet of Tablets.
+It requires a Zookeeper ensemble to be running in the same Kubernetes cluster.
+
+
+To start Zookeeper use
+helm install zk bitnami/zookeeper \
+  --set replicaCount=3 \
+  --set auth.enabled=false \
+  --set persistence.size=5Gi
+
```

Comment:
We need to add more contents in Readme to introduce how to build a fluss cluster and operate by fluss in helm charts